crates/db_schema/src/impls/multi_community.rs

```diff
@@ -0,0 +1,84 @@
+use crate::{
+  newtypes::{CommunityId, MultiCommunityId, PersonId},
+  source::multi_community::{MultiCommunity, MultiCommunityInsertForm},
+  utils::{get_conn, DbPool},
+};
+use diesel::{
+  dsl::{delete, insert_into},
+  result::Error,
+  ExpressionMethods,
+  QueryDsl,
+};
+use diesel_async::{scoped_futures::ScopedFutureExt, AsyncConnection, RunQueryDsl};
+use lemmy_db_schema_file::schema::{multi_community, multi_community_entry};
+
+impl MultiCommunity {
+  pub async fn create(
```

Comment:
Would be much better to just implement Crud. Check out `crates/db_schema/src/impls/oauth_provider.rs` for a close example, it should have everything you need.

Reply:
Had that before but it didnt work with transactions (because transactions pass DbConn param but Crud only allows DbPool).